live-examples/css-examples/grid/meta.json

```diff
@@ -0,0 +1,154 @@
+            "title": "CSS Demo: grid-column-end",
+            "type": "css"
+        },
+        "gridRow": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/grid/grid-row.css",
+            "exampleCode":
+                "live-examples/css-examples/grid/grid-row.html",
+            "fileName": "grid-row.html",
+            "title": "CSS Demo: grid-row",
+            "type": "css"
+        },
+        "gridRowStart": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/grid/grid-row-start.css",
+            "exampleCode":
+                "live-examples/css-examples/grid/grid-row-start.html",
+            "fileName": "grid-row-start.html",
+            "title": "CSS Demo: grid-row-start",
+            "type": "css"
+        },
+        "gridRowEnd": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/grid/grid-row-end.css",
+            "exampleCode":
+                "live-examples/css-examples/grid/grid-row-end.html",
+            "fileName": "grid-row-end.html",
+            "title": "CSS Demo: grid-row-end",
+            "type": "css"
+        },
+        "gridAutoColumns": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/grid/grid-auto-columns.css",
+            "exampleCode":
+                "live-examples/css-examples/grid/grid-auto-columns.html",
+            "fileName": "grid-auto-columns.html",
+            "title": "CSS Demo: grid-auto-columns",
+            "type": "css"
+        },
+        "gridAutoRows": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/grid/grid-auto-rows.css",
+            "exampleCode":
+                "live-examples/css-examples/grid/grid-auto-rows.html",
+            "fileName": "grid-auto-rows.html",
+            "title": "CSS Demo: grid-auto-rows",
+            "type": "css"
+        },
+        "gridAutoFlow": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/grid/grid-auto-flow.css",
+            "exampleCode":
+                "live-examples/css-examples/grid/grid-auto-flow.html",
+            "fileName": "grid-auto-flow.html",
+            "title": "CSS Demo: grid-auto-flow",
+            "type": "css"
+        },
+        "repeat": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/grid/function-repeat.css",
+            "exampleCode":
+                "live-examples/css-examples/grid/function-repeat.html",
+            "fileName": "function-repeat.html",
+            "title": "CSS Demo: repeat",
```

Comment:
We should include brackets for functions: `CSS Demo: repeat()`, to match the title.